icons/stroke_ipld.svg

```diff
@@ -1,2 +1 @@
304 4.304 0 0 0-1.269 3.063c0 1.157.45 2.245 1.269 3.062a4.282 4.282 0 0 0 1.963 1.107v2.391L51.236 48.627v-8.901l9.304-5.877v-7.051a4.282 4.282 0 0 0 1.986-1.114 4.336 4.336 0 0 0-.001-6.125 4.337 4.337 0 0 0-6.125 0h-.001a4.337 4.337 0 0 0 0 6.125c.52.519 1.145.859 1.804 1.059v5.818l-6.967 4.401V15.561a4.283 4.283 0 0 0 2.008-1.12 4.337 4.337 0 0 0-.001-6.126 4.337 4.337 0 0 0-6.125 0h-.001a4.337 4.337 0 0 0 .001 6.126 4.273 4.273 0 0 0 1.782 1.053v15.881l-7.229-4.566v-3.436a4.292 4.292 0 0 0 2.008-1.12 4.337 4.337 0 0 0 0-6.125 4.337 4.337 0 0 0-6.126 0 4.304 4.304 0 0 0-1.269 3.063c0 1.157.45 2.245 1.269 3.062a4.276 4.276 0 0 0 1.781 1.053v4.79l9.565 6.042v21.108l-7.708-4.226v-7.809a4.275 4.275 0 0 0 2.034-1.128 4.337 4.337 0 0 0 0-6.125 4.337 4.337 0 0 0-6.126 0 4.304 4.304 0 0 0-1.269 3.063c0 1.157.45 2.245 1.269 3.062a4.284 4.284 0 0 0 1.756 1.046v6.61L27.15 43.322v-6.217a4.28 4.28 0 0 0 1.984-1.117 4.3 4.3 0 0 0 1.27-3.063c0-1.157-.45-2.245-1.269-3.062a4.336 4.336 0 0 0-6.126 0 4.337 4.337 0 0 0 0 6.125c.51.51 1.132.858 1.804 1.062v7.621l3.646 2.004v13.632l-10.582-5.77v-2.418a4.29 4.29 0 0 0 2.051-1.133 4.337 4.337 0 0 0 0-6.125 4.304 4.304 0 0 0-3.062-1.269 4.304 4.304 0 0 0-3.063 1.269 4.337 4.337 0 0 0 0 6.125 4.285 4.285 0 0 0 1.738 1.041v3.812L48.9 74.108v11.397a4.277 4.277 0 0 0-1.783 1.053 4.337 4.337 0 0 0 0 6.125 4.32 4.32 0 0 0 3.063 1.266 4.316 4.316 0 0 0 3.063-1.267 4.337 4.337 0 0 0 0-6.125 4.29 4.29 0 0 0-2.008-1.119V73.943L84.82 52.727v-3.892a4.29 4.29 0 0 0 1.625-1.008 4.3 4.3 0 0 0 1.269-3.063zM30.586 61.468V47.844L48.9 57.91v13.542l-18.314-9.984zm34.876-19.065v19.79l-14.227 8.986v-19.79l14.227-8.986zm3.067-12.164c0-.676.264-1.312.742-1.79a2.525 2.525 0 0 1 1.79-.74c.647 0 1.296.247 1.789.74a2.536 2.536 0 0 1 0 3.581 2.535 2.535 0 0 1-3.579 0 2.516 2.516 0 0 1-.742-1.791zm-10.857-9.407a2.534 2.534 0 0 1 3.58 0 2.534 2.534 0 0 1 0 3.58 2.534 2.534 0 0 1-3.58 0 2.534 2.534 0 0 1 0-3.58zM48.389 9.589a2.536 2.536 0 0 1 3.581 0c.987.987.987 2.593.001 3.58a2.536 2.536 0 0 1-3.581 0 2.535 2.535 0 0 1-.001-3.58zM38.084 19.19c0-.676.264-1.312.742-1.79a2.525 2.525 0 0 1 1.79-.74c.647 0 1.296.247 1.789.74a2.536 2.536 0 0 1 0 3.581 2.535 2.535 0 0 1-3.579 0 2.52 2.52 0 0 1-.742-1.791zm-.453 19.832c0-.676.264-1.312.742-1.79a2.525 2.525 0 0 1 1.79-.74c.647 0 1.296.247 1.789.74a2.536 2.536 0 0 1 0 3.581 2.535 2.535 0 0 1-3.58 0 2.525 2.525 0 0 1-.741-1.791zm-13.348-7.887a2.524 2.524 0 0 1 1.789-.74 2.526 2.526 0 0 1 2.532 2.531c0 .677-.264 1.312-.742 1.79-.422.422-.984.647-1.562.696h-.456c-.577-.049-1.139-.274-1.561-.696a2.534 2.534 0 0 1 0-3.581zm-9.207 14.999a2.514 2.514 0 0 1 1.79-.742 2.534 2.534 0 0 1 1.789 4.322 2.533 2.533 0 0 1-3.579 0 2.533 2.533 0 0 1 0-3.58zM51.97 91.411a2.534 2.534 0 0 1-3.58 0 2.534 2.534 0 0 1 0-3.58 2.536 2.536 0 0 1 3.58 0 2.534 2.534 0 0 1 0 3.58zm33.202-44.857c-.955.955-2.624.957-3.579 0a2.534 2.534 0 0 1 1.789-4.322c.677 0 1.312.264 1.791.742.478.478.741 1.113.741 1.79s-.263 1.311-.742 1.79z"/></svg>
```

Comment:
@hacdias did anything change in stroke_ipld.svg ? Can you revert it if not please?

Reply:
@olizilla it didn't change visually. I just run svgo to compress all the files.